bench/README.md

```diff
@@ -65,43 +68,123 @@ And open `http://localhost:9090/` to query and monitor the server-side metrics.
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
 
 To run read load tests
 
-```bash
-locust --headless DCLReadUser
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless ReadModelLoadTest --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless ReadModelLoadTest --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
 
-## Web UI
 
 To run write/read load tests
 
-```bash
-locust
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
+
+
+## Web UI
 
 To run write load tests
 
-```bash
-locust DCLWriteUser
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py WriteModelLoadTest --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py WriteModelLoadTest --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
 
 To run read load tests
 
-```bash
-locust DCLReadUser
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
```

Comment:
Please check that this configuration works without any issues. Because if 2 write users run in one worker they may cause a concurrency issue

Reply:
Done! ✅